storage/innobase/xtrabackup/doc/source/using_xtrabackup/privileges.rst

```diff
@@ -68,7 +70,7 @@ alteration, see :option:`mysql --help` for details.
 
    In case of multiple server instances the correct connection parameters
    (port, socket, host) must be specified in order for |xtrabackup| to talk to
-   the correct server. 
+   the correct server.
 
 
 Permissions and Privileges Needed
```

Comment:
ah... PROCESS was listed already... then we need to change its description
```
PROCESS in order to see all threads which are running on the server (see Improved FLUSH TABLES WITH READ LOCK handling),
```
to `SHOW ENGINE INNODB STATUS`